cpp/modules/luma.gl/core/src/glfw-animation-loop.cpp

```diff
@@ -40,7 +40,9 @@
 using namespace lumagl;
 using namespace lumagl::utils;
 
-GLFWAnimationLoop::GLFWAnimationLoop(const wgpu::BackendType backendType, std::shared_ptr<wgpu::Device> device) {
+GLFWAnimationLoop::GLFWAnimationLoop(const wgpu::BackendType backendType, std::shared_ptr<wgpu::Device> device,
+                                     int width, int height)
```

Comment:
We should probably have an `Options` parameter here, inheriting from animation loop options. But this is good enough for now.